stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetAddCardFragment.kt

```diff
@@ -52,5 +53,11 @@ internal class PaymentSheetAddCardFragment(
                 sheetViewModel.checkout()
             }
         }
+
+        sheetViewModel.viewState.observe(viewLifecycleOwner) { viewState ->
```

Comment:
Is this still needed given the changes in: https://github.com/stripe/stripe-android/pull/3522.  Perhaps you can help describe in which case this is needed.

Reply:
This is needed to set `selection` back to the card being entered by the user. Otherwise it'll still be Google Pay, and calling `checkout` would start Google Pay checkout.

Reply:
ah, very nice.